partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,33 @@
+            )
+            self._create(config)
+
+    async def instantiate(self, part):
+        await super().instantiate(part)
+
+        with pc_logging.Action("BREP", part.project_name, part.name):
+            shape = TopoDS_Shape()
+            builder = BRep_Builder()
+
+            if not BRepTools().Read_s(shape, self.path, builder):
+                raise ValueError(f"Failed to load BREP file: {self.path}")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)